ServerCore/Pages/Teams/Details.cshtml

```diff
@@ -10,6 +10,30 @@
 }
 @using Microsoft.AspNetCore.Http.Extensions @* for GetEncodedUrl *@
 
+@{
+    var mailtoUrl = "mailto:";
```

Comment:
Could this logic be under the disqualification check so we don't have to build the email for the vast majority of cases where the team isn't disqualified?